src/renderer/components/SettingsView/index.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react'
+import InfoIcon from '@material-ui/icons/InfoOutlined'
+import { SettingsMenu } from './SettingsMenu'
+import { defineMessages } from 'react-intl'
+import { AboutMapeoMenu } from './AboutMapeo'
+import styled from 'styled-components'
+
+const m = defineMessages({
+  aboutMapeo: 'About Mapeo',
+  aboutMapeoSubtitle: 'Version and build number'
+})
+
+const tabs = /** @typedef {const} */ [
```

Comment:
And then we want to define this as

`/** @type {import('./SettingsMenu')tab[]} */`

This means `tabId` can only equal the tabs that we set, instead of a generic string